### DISCUSSION TITLE
Bestandsverzeichnis

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#70] [Shipped - Live on production]
Aktuell gibt es kein Bestandsverzeichnis nach §13 MPBetreibV.

https://www.gesetze-im-internet.de/mpbetreibv/__13.html

Diese Liste ist in DE essenziell und muss von der Einrichtung geführt werden. Da der Excel Export der Geräteliste nur das enthält, was gerade als Spalten angezeigt wird und auch nicht alle Infos anzeigbar sind, kann man den nur bedingt als Bestandsverzeichnis nutzen.

Vlt. kann man im Grid **Gerätedaten** unter **Export** neben Excel noch eine zweite Option mit **Bestandsverzeichnis** anbieten, die immer die fixen und geforderten Infos enthält

**Comment by neongrau** [comment]
Was ist denn mit unserem MP Buch PDF?

Aber ja eine XLSX Variante lässt sich sicher machen.

**Comment by YvesR** [comment]
Verstehe ich das richtig, statt dem PDF 1:1 die gleichen Infos, aber als Excel Datei?

**Reply by moritzvolkmar** [reply]
Nein, ein Bestandsverzeichnis enthält einfach fix folgende Daten und ist eine Liste aller aktiven Medizinprodukte in der Einrichtung. Das MP Buch ist ja nur für ein einzelnes Gerät. Das Format ist dabei egal. Also theoretisch auch einfach ein fixes Grid, das man exportieren kann. Oder halt ein PDF Report. Es muss aber folgende Infos enthalten:

- [ ] Gerätetyp & Geräteart
- [ ] Hersteller + dessen Anschrift (Adresse haben wir aktuell nicht, kenne aber auch niemanden der das aktuell wirklich im Bestandsverzeichnis aufführt)
- [ ] Seriennummer
- [ ] Anschaffungsjahr (theoretisch das Inbetriebnahmedatum oder besser das Kaufdatum)
- [ ] CE Kennzeichen + Nr. der Benannten Stelle (bspw. CE 1234)
- [ ] Inventarnummer
- [ ] Gerätestandort (laut Gesetz ist auch der Name der Einrichtung ausreichend)
- [ ] STK Intervall


Gesetzestext:

1. Der Betreiber hat für alle aktiven nichtimplantierbaren Medizinprodukte der jeweiligen Betriebsstätte ein Bestandsverzeichnis zu führen. Die Aufnahme in ein Verzeichnis, das auf Grund anderer Vorschriften geführt wird, ist zulässig.
2.  In das Bestandsverzeichnis sind für jedes Medizinprodukt nach Absatz 1 folgende Angaben einzutragen:

- Bezeichnung, Art und Typ, Loscode oder die Seriennummer, Anschaffungsjahr des Medizinproduktes,
- Name oder Firma **und** die Anschrift des Herstellers oder des Bevollmächtigten oder, sofern der Hersteller keinen Unternehmenssitz in der Europäischen Union und keinen Bevollmächtigten beauftragt hat, des Importeurs,
- die der CE-Kennzeichnung hinzugefügte Kennnummer der Benannten Stelle, soweit diese nach den Vorschriften des Medizinproduktegesetzes in der bis einschließlich 25. Mai 2021 geltenden Fassung oder nach der Verordnung (EU) 2017/745 angegeben ist,
- soweit vorhanden, betriebliche Identifikationsnummer,
- Standort und betriebliche Zuordnung,
- die nach § 11 Absatz 1 Satz 2 und 3 festgelegte Frist für sicherheitstechnische Kontrollen.

3. Für das Bestandsverzeichnis sind alle Datenträger zulässig, sofern die Angaben nach Absatz 2 Satz 1 innerhalb einer angemessenen Frist lesbar gemacht werden können.

**Reply by YvesR** [reply]
@moritzvolkmar 
We will implement in bi.samedis.care as `Regulatory inventory`. IMHO the inventory list inside the app is enough, but if the German bureaucracy shows up... never mind :).

Some notes:
- We can not show STK, as we have no clue per inventory, so we will take all service intervals with dimension month or year
- We can not CE, as we have regulatory key value pairs, so we will show all existing for this device-mode
